### PR TITLE
Lenka's part

### PR DESCRIPTION
**Pull request by LenkaCh**
what shall i do?

### CODE REVIEW

**Comment by KristinaPetrle** [comment]
cau lenka, good job

**Comment by quentingaeschlin** [comment]
Lenka, pls amend your lenka file. That's not even a haiku...

**Comment by juraj-prachar** [comment]
write bye bye to a new line

File: branch_training/Lenka [hunk 1]
```diff
@@ -0,0 +1,2 @@
+ahg
```
Comment (quentingaeschlin) [inline branch_training/Lenka]:
```suggestion
ahg he expired
all this pain
so little gain
```

**Review by quentingaeschlin — CHANGES_REQUESTED** [review]

**Comment by Ladislav1976** [comment]
Lenka, great job! 👍

**Review by quentingaeschlin — APPROVED** [review]

**Pull request merged**